src/commonTest/kotlin/kotools/types/number/PositiveIntTest.kt

```diff
@@ -5,13 +5,13 @@ import kotlinx.serialization.KSerializer
 import kotlinx.serialization.SerializationException
 import kotlinx.serialization.encodeToString
 import kotlinx.serialization.json.Json
+import kotools.types.*
```

Comment:
Why there is a wildcard import with other classic ones?
We usually avoid to add wildcard imports in this project for facilitating the review of requirements.